### PR TITLE
Fasih/DPROD-3003/Implement message or alert section

### PR DESCRIPTION
**Pull request by fasihali-deriv**
Implement message information section

Screenshot 2024-03-15 at 5 05 28 PM

### CODE REVIEW

File: src/components/section-message/information/index.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          <div>
+            <LabelPairedCircleInfoBoldIcon
+              iconSize="sm"
+              className="fill-status-info"
+              data-testid="dt-info-icon"
+            />
+          </div>
+          <div
+            className={qtMerge(
+              'flex flex-col',
```
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:54]:
```suggestion
            className={qtJoin(
              'flex flex-col',
```

File: src/components/section-message/information/index.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              (title || description) && 'gap-gap-md',
+            )}
+          >
+            <div>
+              {title && (
+                <Text className={qtMerge(SectionMessageTitleSizeCVA({ size }))}>
+                  {title}
```
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:61]: here as well

File: src/components/section-message/information/index.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                </Text>
+              )}
+              {description && (
+                <Text
+                  className={qtMerge(
+                    qtMerge(SectionMessageMessageSizeCVA({ size })),
```
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:67]: here as well

File: src/components/section-message/information/index.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+              )}
+            </div>
+            <div className="flex items-start justify-start gap-gap-md">
+              {link1 && (
+                <Link
+                  {...link1}
+                  className={qtMerge(
+                    SectionMessageLinkSizeCVA({ size }),
+                    'p-50 text-solid-slate-1400',
+                  )}
```
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:82]: here as well

File: src/components/section-message/information/index.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                  size={size}
+                />
+              )}
+              {link2 && (
+                <Link
+                  {...link2}
+                  className={qtMerge(
+                    SectionMessageLinkSizeCVA({ size }),
+                    'p-50 text-solid-slate-1400',
+                  )}
```
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:92]: here as well

File: src/components/section-message/information/index.tsx (hunk 3 above)
Comment (hasan-deriv) [inline src/components/section-message/information/index.tsx:67]:
```suggestion
                    qtMerge(SectionMessageDescriptionSizeCVA({ size })),
```

**Review by hasan-deriv — CHANGES_REQUESTED** [review]

File: src/components/section-message/information/index.tsx (hunk 5 above)
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: Done

File: src/components/section-message/information/index.tsx (hunk 4 above)
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: Done

File: src/components/section-message/information/index.tsx (hunk 3 above)
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: Fixed

Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: Fixed

File: src/components/section-message/information/index.tsx (hunk 2 above)
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: fixed

File: src/components/section-message/information/index.tsx (hunk 1 above)
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: fixed

File: src/components/section-message/information/index.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        data-testid="dt-info-component"
+      >
+        <div className="flex items-start gap-gap-md">
+          <div>
+            <LabelPairedCircleInfoBoldIcon
+              iconSize="sm"
+              className="fill-status-info"
+              data-testid="dt-info-icon"
+            />
+          </div>
```
[the remark this answers is not in the document]
Comment (fasihali-deriv) [reply src/components/section-message/information/index.tsx]: Thanks @hasan-deriv for raising this. 
I was only able to remove one div.

**Review by hasan-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**